daemon/algod/api/server/v2/handlers.go

```diff
@@ -67,11 +67,18 @@ type LedgerForAPI interface {
 	Latest() basics.Round
 	LookupAsset(rnd basics.Round, addr basics.Address, aidx basics.AssetIndex) (ledgercore.AssetResource, error)
 	LookupApplication(rnd basics.Round, addr basics.Address, aidx basics.AppIndex) (ledgercore.AppResource, error)
+	LookupWithoutRewards(rnd basics.Round, addr basics.Address) (ledgercore.AccountData, basics.Round, error)
+	CheckDup(currentProto config.ConsensusParams, current basics.Round, firstValid basics.Round, lastValid basics.Round, txid transactions.Txid, txl ledgercore.Txlease) error
 	BlockCert(rnd basics.Round) (blk bookkeeping.Block, cert agreement.Certificate, err error)
+	GenesisHash() crypto.Digest
```

Comment:
@jdtzmn and reviewers - Leaving a question to further my understanding.  If easier to address live, let me know.

Is it preferable to expand `LedgerForAPI` rather than expose a way to retrieve `LedgerForEvaluator`?

* I understand it's _not_ possible as is to expose `LedgerForEvaluator` in v2/handlers.go due to _internal_ namespacing.
* I'm trying to gauge if the thinking is either (or something else):
  * It's inappropriate to expose `LedgerForEvaluator`, and it's rightfully protected by _internal_.
  * _internal_ usage is a historical artifact that no longer applies and _if_ moved, we'd prefer to stop expanding `LedgerForAPI`.

In part, my question may also touch on how to model interfaces in Go.  I suspect my understanding is incomplete.

Reply:
My opinion: The `internal` thing was a mistake. I believe that is a widely held opinion and I expect it will be reverted soon.

If the API needs to evaluate transactions, it does not seem strange that it would need an interface that's as wide as LedgerForEvaluator.

Reply:
@jannotti Got it - thanks for the response.  Good to resolve on my end.